wiki/reverse-proxy.md

```diff
@@ -34,12 +33,14 @@ Apache virtual server configuration
         ProxyRequests           Off
         ProxyPreserveHost       On
         AllowEncodedSlashes     NoDecode
-        ProxyPass               /       http://127.0.0.1:4873 nocanon
-        ProxyPassReverse        /       http://127.0.0.1:4873
+        ProxyPass               /       http://127.0.0.1:4873/ nocanon
+        ProxyPassReverse        /       http://127.0.0.1:4873/
```

Comment:
I'm confused about this. Just here https://github.com/WickedSik/verdaccio/commit/21df66d6225b5b7ab8655d3c69f7656351bfe54c#diff-b87924c43dc08aa873f812fd5cd8af94 was removed those trailing slash. I think we need to define better when those trailing slash are required or not.